stdlib/select.pyi

```diff
@@ -137,7 +137,6 @@ if sys.platform == "linux":
     EPOLLRDNORM: int
     EPOLLWRBAND: int
     EPOLLWRNORM: int
-    EPOLL_RDHUP: int
```

Comment:
I don't think this ever existed at runtime. I think it was just a figment of our imagination. Note the name is very similar to `select.EPOLLRDHUP`, which _does_ exist at runtime (and which we already have elsewhere in the stub).